src/System.ComponentModel.TypeConverter/src/System/ComponentModel/AttributeCollection.cs

```diff
@@ -5,7 +5,6 @@
 using System.Reflection;
 using System.Collections;
 
-[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Security", "CA2112:SecuredTypesShouldNotExposeFields", Scope = "type", Target = "System.ComponentModel.AttributeCollection")]
```

Comment:
I removed this because it relates to LinkDemands, which is CAS. Does it have another purpose?

`A public or protected type contains public fields and is secured by a Link Demands.`